bindings/ruby/lib/typelib/clang.rb

```diff
@@ -69,6 +69,20 @@ def self.load(registry, file, kind, options)
             end
         end
 
+        def self.clang_binary_name
+            if !`which clang-3.4`.empty?
```

Comment:
Please redirect the stderr of which to /dev/null, otherwise the "blabla not found" error message pollutes the output